python-lib/plugin_io_utils.py

```diff
@@ -97,6 +106,16 @@ def move_columns_after(
     return df.reindex(columns=reordered_columns)
 
 
+def get_keyword(text: AnyStr, case_insensitive: bool) -> AnyStr:
```

Comment:
Not sure a function is required for this... Is this function used in a lot of places in your code?

Reply:
Yes, and I've planned to use the same function for diacritic removal, so it will get a little bit bigger in the next PR

Reply:
Okay - in that case, can you keep it but move it as a static method to one of your NLP-related modules? The plugin_io_utils module should be for generic io stuff only.

Reply:
Actually, the method is called on both Tagger and Formatter. I've put it with get_sentence in a nlp_utils.py file instead in this commit : 89f22ca. Are you okay with that solution? :)

Reply:
perfect!